scene/gui/tree.cpp

```diff
@@ -4555,7 +4555,7 @@ void Tree::ensure_cursor_is_visible() {
 		y_offset -= tbh;
 
 		const int cell_h = compute_item_height(selected_item) + theme_cache.vseparation;
-		const int screen_h = area_size.height - h_scroll->get_combined_minimum_size().height - tbh;
+		const int screen_h = area_size.height - h_scroll->is_visible() * h_scroll->get_combined_minimum_size().height - tbh;
```

Comment:
This it a bit of a weird trick. I think it would be a bit cleaner to avoid multiplying a bool by an int here.
If you want to keep things in line, a ternary operator would be better IMO: 
`const int screen_h = area_size.height - (h_scroll->is_visible() ? h_scroll->get_combined_minimum_size().height : 0) - tbh;

Reply:
I always see bool as very small int :)
I will correct this.

Reply:
IMO explicit `int(h_scroll->is_visible())` is ok too.